lib/api/objectGetACL.js

```diff
@@ -1,11 +1,15 @@
 import { errors } from 'arsenal';
+import async from 'async';
```

Comment:
Please move this above arsenal. We generally follow the convention core node.js modules > third party modules > scality modules > local modules.